src/main/java/com/coursemngsys/coursemanagementsystem/temp/User.java

```diff
@@ -0,0 +1,58 @@
+package com.coursemngsys.coursemanagementsystem.temp;
+
+import java.util.ArrayList;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+/**
+ *
+ * @author 56130
+ */
+public class User {
+
+    private int id ;
```

Comment:
Nereikalingas tarpas.
```suggestion
    private int id;
```

Reply:
Tikėtina, kad "id" nebus keičiamas, nurodytos konstantos pagerina kodo skaitomumą.
```suggestion
    private final int id ;
```